layouts/shortcodes/demo_description.md

```diff
@@ -0,0 +1,24 @@
+The [official OpenTelemetry
+Demo](https://github.com/open-telemetry/opentelemetry-demo) is a reference
+application showing how to use OpenTelemetry in a real-world environment.
+
+The demo does the following:
+
+* Provide a realistic example of a distributed system that can be used to
+  demonstrate OpenTelemetry instrumentation and observability.
+* Build a base for vendors, tooling authors, and others to extend and
+  demonstrate their OpenTelemetry integrations.
+* Create a living example for OpenTelemetry contributors to use for testing new
+  versions of the API, SDK, and other components or enhancements.
+
+If you find yourself asking questions like:
+
+* How do I consider the
```

Comment:
I'd move this bullet to the last position because it's probably the most complex question on the list.